modules/govuk/manifests/apps/email_status_updates.pp

```diff
@@ -0,0 +1,30 @@
+class govuk::apps::email_status_updates {
+  $port = '3088'
+  $app_domain = hiera('app_domain')
+
+  nginx::config::site { "email-status-updates.${app_domain}":
+    content => "
+      upstream email-status-updates.${app_domain}-proxy {
+        server localhost:${port};
+      }
+
+      server {
+        server_name email-status-updates.${app_domain};
+
+        listen 80;
+        listen 443 ssl;
+
+        ssl_certificate /etc/nginx/ssl/email-alert-api.${app_domain}.crt;
+        ssl_certificate_key /etc/nginx/ssl/email-alert-api.${app_domain}.key;
+        include /etc/nginx/ssl.conf;
+
+        add_header Access-Control-Allow-Origin *;
+        add_header Access-Control-Allow-Methods POST;
+
+        location / {
+          proxy_pass http://email-status-updates.${app_domain}-proxy/status_updates;
```

Comment:
Should this be proxying to `email-alert-api`?

Reply:
It is, but it's a bit confusing.

The `-proxy` upstream is defined above (lines 7 through 9), which sends traffic to `locahost:3088`.

We might be able to re-use `email-alert-api` as the upstreams as it's defined by `email_alert_api.pp`, but we'd be slightly coupled across Nginx config.

Reply:
Ah right yes, I always get confused by these things that look like URLs but are actually references to a proxy config...